anytask/manage.py

```diff
@@ -23,4 +22,8 @@
 sys.modules['django.contrib.auth.tests'] = None
 
 if __name__ == "__main__":
-    execute_manager(settings)
+    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "anytask.settings")
```

Comment:
```suggestion
    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "settings")
```
Скорее всего тут так надо.